automatic/nuget.commandline/update.ps1

```diff
@@ -19,21 +19,30 @@ function global:au_BeforeUpdate {
 function global:au_GetLatest {
   $json = Invoke-WebRequest -UseBasicParsing -Uri $releases | ConvertFrom-Json
 
-  $versions = $json."nuget.exe" | Sort-Object uploaded -Descending
-  $preRelease = $versions | ? stage -EQ 'EarlyAccessPreview' | select -First 1
-  $stableRelease = $versions | ? stage -EQ 'ReleasedAndBlessed' | select -First 1
```

Comment:
This logic seems to be lost with your changes. Resulting in that also versions in stage `Released` will also be published. But this would be the same logic agains as before https://github.com/chocolatey-community/chocolatey-coreteampackages/commit/439567167d90cdbfcf696161e7de4af6044bce92#diff-25fdf79c9a692cdfad0afe85cbcb1e9f, which is also what is documented in the [README Notes](https://github.com/chocolatey-community/chocolatey-coreteampackages/blob/master/automatic/nuget.commandline/README.md). So I'm OK with this change

Reply:
It is true, the logic more or less reverts back to that commit.
The reason (from my perspective) for it to only check the EarlyAccessPreview and the ReleasedAndBlessed, was because of when I made the changes back then those two was the only ones listed in the json file.